apps/www/components/Track.js

```diff
@@ -19,7 +19,14 @@ const trackRoles = (me) =>
     me ? [].concat(me.roles).sort().join(' ') || 'none' : 'guest',
   ])
 
+let lastTrackedUrl
```

Comment:
Any particular reason this code is outside of the component?

Reply:
don't want to use a react ref and I prefer placing functions outside of react components whenever possible to avoid shadowing issues